migrations/blocked_tasks.js

```diff
@@ -0,0 +1,83 @@
+// constants
+finishedStatuses = ["success","failed"]
+
+function taskIsBlocked(task) {
+    for(dep of task.depends_on) {
+        if (dep.unattainable) {
+            return true
+        }
+    }
+
+    return false
+}
+
+function unattainableResolved(task) {
+    for (dep of task.depends_on) {
+        if (dep.unattainable == null) {
+            return false
+        }
+    }
+    return true
+}
+
+//
+// example invocation:
+//
+// startingDate = ISODate("2016-01-01T00:00:00Z")
```

Comment:
what happens to older things?

Reply:
the starting date needs to be set to the earliest relevant starting point. Maybe 06-01-2013? (there are 9 tasks with dependencies that will need to be addressed separately since their create_time is set to Go's zero time)